entity-type-creator/pages/api/socket.ts

```diff
@@ -134,26 +134,84 @@ export default function SocketHandler(req: NextApiRequest, res: NextApiResponse<
       socket.emit('translations', JSON.parse((await readFile('../translations/mod-fqm-manager/en.json')).toString()));
     });
 
-    socket.on('update-translations', async (newTranslations: Record<string, string>) => {
-      if (Object.keys(newTranslations).length === 0) return;
+    socket.on(
```

Comment:
this diff looks disgusting. main change is that it checks which keys are expected, removing unused ones and telling you if any are missing